src/pages/messages/TasksPage.tsx

```diff
@@ -85,6 +85,9 @@ const TasksPage = () => {
     meta: { errorMessage: t('feedback.error.get_person') },
   });
 
+  const urlSearchQuery = new URLSearchParams(location.search).get('query');
+  const searchQuery = urlSearchQuery ? `&query=${urlSearchQuery}` : '';
```

Comment:
ser litt feil ut dette med å prefikse med `&` og etterpå joine med `AND`... skal undersøke

Reply:
fixed. litt ulik struktur for søket til NVI og tickets, så det må håndter ulikt